com.kolor.AutopanoPro.yml

```diff
@@ -0,0 +1,53 @@
+app-id: com.kolor.AutopanoPro
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-version: "5.15-21.08"
+
+rename-icon: AutopanoPro
+rename-desktop-file: AutopanoPro.desktop
+command: AutopanoPro
+
+tags:
+    - proprietary
+finish-args:
+    - --share=ipc
+    - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
I don't know, does it cause harm? Should I remove it?

Reply:
since you use the fallback, you should know by now if you tried the flatpak. But given it's proprietary and obsolete, I wonder.